### ISSUE TITLE
Laplace transform assertion failure

### ISSUE THREAD

**Issue by sympy-issue-migrator** [sympy/sympy#6311]
```
Playing with the laplace_transform capabilities, I
found that it does not handle some shifts correctly.
In the following example, an assertion fails, and it
is clearly because of the shift.

from sympy.integrals.transforms import *
L = laplace_transform
iL = inverse_laplace_transform
s=Symbol('s')
t=Symbol('t')
L(Heaviside(t)*cos(t),t,s)
L(Heaviside(t-1)*cos(t-1),t,s)

This seems to be related to trig term not being coverted
to the correct exponential first?  The code is doing everything
in terms of Mellon transforms, which is completely outside my
standard approach to these problems.  No idea what the best way 
is to approach fixing the issue.
```

Original issue for #6311: http://code.google.com/p/sympy/issues/detail?id=3212
Original author: https://code.google.com/u/112862214195610862534/

**Comment by hanspi42** [comment]
> 
> 
> This is fixed in #2737

So the issue could be closed?

**Comment by oscarbenjamin** [comment]
I get this:
```python
In [3]: L(Heaviside(t)*cos(t),t,s)
Out[3]: 
⎛  s            ⎞
⎜──────, 0, True⎟
⎜ 2             ⎟
⎝s  + 1         ⎠

In [4]: L(Heaviside(t-1)*cos(t-1),t,s)
Out[4]: LaplaceTransform(cos(t - 1)⋅θ(t - 1), t, s)
```
What should the correct answer be for `Out[4]`? I guess the issue can be closed when that is handled.

**Comment by hanspi42** [comment]
This has long been solved, it was already OK in SymPy 1.10.1.

The issue can be closed.
